frontend/src/components/Cart.jsx

```
@@ -0,0 +1,20 @@
+
+const Cart = () => {
+  const cart = useSelector((state) => state?.cart?.value?.cart);
+  return (
+    <div className="items-center flex gap-x-6">
+      <i className="nf nf-fa-search text-lg"></i>
+      <i className="nf nf-fa-bag_shopping relative text-lg">
+        <Badge className={"absolute left-1/2 -top-1/2 text-xs"}>
+          {cart?.length || 0}
+        </Badge>
+      </i>
+      <i className="nf nf-md-signal -rotate-90 md:hidden"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)